msg/messages.js

```diff
@@ -20,1220 +20,275 @@
 
 /**
  * @fileoverview English strings.
- * @author fraser@google.com (Neil Fraser)
+ * @author ascii@media.mit.edu (Andrew Sliwinski)
  *
- * After modifying this file, either run "build.py" from the parent directory,
- * or run (from this directory):
- * ../i18n/js_to_json.py
- * to regenerate json/{en,qqq,synonyms}.json.
- *
- * To convert all of the json files to .js files, run:
- * ../i18n/create_messages.py json/*.json
+ * After modifying this file, run `npm run translate` from the root directory
+ * to regenerate `./msg/json/en.json`.
  */
 'use strict';
 
 goog.provide('Blockly.Msg.en');
-
 goog.require('Blockly.Msg');
 
-
-/**
- * Due to the frequency of long strings, the 80-column wrap rule need not apply
- * to message files.
- */
-
-/**
- * Each message is preceded with a tripple-slash comment that becomes the
- * message descriptor.  The build process extracts these descriptors, adds
- * them to msg/json/qqq.json, and they show up in the translation console.
- */
-
-/// {{Notranslate}} Hue value for all logic blocks.
-Blockly.Msg.LOGIC_HUE = '210';
-/// {{Notranslate}} Hue value for all loop blocks.
-Blockly.Msg.LOOPS_HUE = '120';
-/// {{Notranslate}} Hue value for all math blocks.
-Blockly.Msg.MATH_HUE = '230';
-/// {{Notranslate}} Hue value for all text blocks.
-Blockly.Msg.TEXTS_HUE = '160';
-/// {{Notranslate}} Hue value for all list blocks.
-Blockly.Msg.LISTS_HUE = '260';
-/// {{Notranslate}} Hue value for all colour blocks.
-Blockly.Msg.COLOUR_HUE = '20';
-/// {{Notranslate}} Hue value for all variable blocks.
-Blockly.Msg.VARIABLES_HUE = '330';
-/// {{Notranslate}} Hue value for all procedure blocks.
-Blockly.Msg.PROCEDURES_HUE = '290';
-
-/// default name - A simple, general default name for a variable, preferably short.
-/// For more context, see
-/// [[Translating:Blockly#infrequent_message_types]].\n{{Identical|Item}}
-Blockly.Msg.VARIABLES_DEFAULT_NAME = 'item';
-/// button text - Button that sets a calendar to today's date.\n{{Identical|Today}}
-Blockly.Msg.TODAY = 'Today';
-
-// Context menus.
-/// context menu - Make a copy of the selected block (and any blocks it contains).\n{{Identical|Duplicate}}
+// Control blocks
+Blockly.Msg.CONTROL_FOREVER = "forever";
```

Comment:
Where possible I've tried to match the opcode (`CATEGORY_OPCODE`). For menu items I'm using the convention of `CATEGORY_OPCODE_MENUITEM`.